clusterman/aws/markets.py

```diff
@@ -394,10 +398,12 @@ def fetch_instance_type_from_aws(instance_type: str) -> InstanceResources:
     disk_size = res.get("InstanceStorageInfo", {}).get("TotalSizeInGB", None)
     gpu_size = res.get("GpuInfo", {}).get("Gpus", [{}])[0].get("Count", 0)
 
+    logger.info(f"located instance-type details: {vcpu_count}, {mem_size}, {disk_size}, {gpu_size}")
     return InstanceResources(vcpu_count, mem_size, disk_size, gpu_size)
 
 
 def get_instance_type(instance_type: str) -> InstanceResources:
+    logger.info(f"fetching instance type details: {instance_type}")
```

Comment:
it will work for all requests not only for getting from aws. is it fine?

Reply:
yeah, I wanted to check for both cases. Right now there are limited logs in this code flow, so couldn't verify the working.